torch/nested/_internal/ops.py

```diff
@@ -1124,6 +1124,11 @@ def native_layer_norm_default(func, *args, **kwargs):
         "normalized_shape"
     ]  # denote -1 as the ragged dimension, i.e. normalize on the ragged dimension
 
+    if len(normalized_shape) < 2:
+        raise RuntimeError(
+            "layer_norm(): not supported for a normalized shape of length less than 2"
+        )
```

Comment:
I think this breaks behavior that previously worked, right? `len(normalized_shape) == 1` seems to work when I test locally before this PR